### DISCUSSION TITLE
Excellent work Elvus2586 if your pull [Sep 29]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3966] [Discord Threads]
> 🙋‍♂️ **emdax**

**Comment by o1lab** [comment]
> 🙋‍♂️ **emdax**  
#### https://github.com/nocodb/nocodb/pull/3588  
#  
> 🙋‍♂️ **emdax**  
#### this functionality is key  
#  
> 🙋‍♂️ **emdax**  
####   
![image0](https://cdn.discordapp.com/attachments/1022273746888839258/1022541461071343726/Screen_Shot_2022-09-20_at_1.44.20_PM.png)  
#  
> 🙋‍♂️ **emdax**  
#### <@&682859083799199814> <@858443413208301611>  super excited about this feature merge  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1022273746888839258)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
